src/device_peak_find_kernel.cu

```diff
@@ -79,13 +79,12 @@ __global__ void dilate_peak_find(const float *d_input, ushort* d_input_taps,  un
 	if(peak==1){
 		list_pos=atomicAdd(gmem_pos, 1);
 		if(list_pos<max_peak_size){
-			d_peak_list_DM[list_pos]  = idxY + shift; // DM coordinate (y)
-			d_peak_list_TS[list_pos]  = idxX*DIT_value + d_input_taps[idxY*width+idxX]/2; // time coordinate (x)
-			d_peak_list_SNR[list_pos] = my_value; // SNR value
-			d_peak_list_BW[list_pos]  = d_input_taps[idxY*width+idxX]; // width of the boxcar
+			d_peak_list[4*list_pos]   = (idxY + shift)*dm_step + dm_low; // DM coordinate (y)
```

Comment:
The old version (left on diff on github) should be kept since it means that no conversion of coordinates happens

Reply:
Will return to the old version - was a bit confused with which one to keep.